modules/exploits/linux/http/opennms_horizon_authenticated_rce.rb

```diff
@@ -0,0 +1,924 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
```

Comment:
With no call to `execute_cmdstager`, I don't think you need this since you're not using the cmdstager.

Reply:
Good point! I initially planned to use one but in the end had no need for it and then forgot to remove this

Reply:
fixed